agent/api/statechange.go

```diff
@@ -144,10 +146,10 @@ func NewContainerStateChangeEvent(task *apitask.Task, cont *apicontainer.Contain
 		Status:        contKnownStatus.BackendStatus(cont.GetSteadyStateStatus()),
 		ExitCode:      cont.GetKnownExitCode(),
 		PortBindings:  cont.GetKnownPortBindings(),
+		ImageDigest:   cont.ImageDigest,
```

Comment:
let's use `GetImageDigest()` with a lock, just in case.

Reply:
cool, will change it.